### ISSUE TITLE
Broken urls for images smaller than requested sizes

### ISSUE THREAD

**Issue by himynameisjonas** [11ty/eleventy-img#4]
While trying out the responsive image based on the `Output Optimized Multi-Format, Multi-Size Responsive Images using <picture>` example in the Readme I noticed that if the original is smaller than the requested widths it will generate url:s that doesn't match the output files.

#### Options:
```js
{
  widths: [500, 1000, 2000, 3000, 4000],
  outputDir: "_site/img",
  formats: ["jpeg"],
};
```
#### Output
```js
{
  jpeg: [
    {
      format: 'jpeg',
      width: 500,
      height: 500,
      url: '/img/22a3e283-500.jpeg',
      sourceType: 'image/jpeg',
      srcset: '/img/22a3e283-500.jpeg 500w',
      outputPath: '_site/img/22a3e283-500.jpeg',
      size: 31094
    },
    {
      format: 'jpeg',
      width: 1000,
      height: 1000,
      url: '/img/22a3e283-1000.jpeg',
      sourceType: 'image/jpeg',
      srcset: '/img/22a3e283-1000.jpeg 1000w',
      outputPath: '_site/img/22a3e283-1000.jpeg',
      size: 148677
    },
    {
      format: 'jpeg',
      width: 1024,
      height: 1024,
      url: '/img/22a3e283-1024.jpeg',
      sourceType: 'image/jpeg',
      srcset: '/img/22a3e283-1024.jpeg 1024w',
      outputPath: '_site/img/22a3e283-2000.jpeg',
      size: 182066
    },
    {
      format: 'jpeg',
      width: 1024,
      height: 1024,
      url: '/img/22a3e283-1024.jpeg',
      sourceType: 'image/jpeg',
      srcset: '/img/22a3e283-1024.jpeg 1024w',
      outputPath: '_site/img/22a3e283-3000.jpeg',
      size: 182066
    },
    {
      format: 'jpeg',
      width: 1024,
      height: 1024,
      url: '/img/22a3e283-1024.jpeg',
      sourceType: 'image/jpeg',
      srcset: '/img/22a3e283-1024.jpeg 1024w',
      outputPath: '_site/img/22a3e283-4000.jpeg',
      size: 182066
    }
  ]
}
```

Notices that the output's url doesn't match the outputPath as soon as the requested size is larger than the original (for example `url: '/img/22a3e283-1024.jpeg', outputPath: '_site/img/22a3e283-4000.jpeg'`)

**Comment by danielpost** [comment]
I just ran into the same issue. Looks like the output filename is written [here](https://github.com/11ty/eleventy-img/blob/master/img.js#L125), without looking at the output from `sharp`.

**Comment by zachleat** [comment]
This will be included with 0.2.5 going out shortly.

**Comment by zachleat** [comment]
0.2.5 is released!
